src/playground/Widgets/TasksList/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Circle, InProgressTask } from "@/icons"
+import { useMemo } from "react"
+
+type TaskStatus = "in-progress" | "due" | "no-due"
+
+type TaskItemProps = {
+  title: string
+  status: TaskStatus
+}
+
+function TaskItem({ title, status }: TaskItemProps) {
+  return (
+    <div className="flex flex-row items-start gap-2">
+      {(status === "due" || status === "no-due") && (
+        <Circle width={24} opacity={status === "no-due" ? 0.4 : 1} />
+      )}
+      {status === "in-progress" && <InProgressTask />}
+      <p className="mt-[2px] line-clamp-2 flex-1 font-medium">{title}</p>
+    </div>
+  )
+}
+
+interface Props {
+  inProgressTasks: string[]
+  noDueTasks: string[]
+  dueTasks: string[]
+  maxTasksToShow?: number
+}
+
+export function TasksList({
```

Comment:
I think we can move this outside of the playground and straight to the main widgets folder. The playground was more for early development, but we are not really supposed to use it at all from Factorial,